composer.json

```diff
@@ -93,9 +93,6 @@
         ]
     },
     "config": {
-        "platform": {
```

Comment:
Don't revert this. This is needed.

Reply:
Is this not forcing 5.5.9 though?

Reply:
Yes, it forces the dependencies to be resolved as if you were 5.5.9. Without this, the lock file will contain dependencies that don't work on PHP 5.5.

Reply:
But does that not conflict with the requirements of PHP 7?

Reply:
No?